src/main/java/net/imglib2/roi/boundary/Boundary.java

```diff
@@ -0,0 +1,296 @@
+package net.imglib2.roi.boundary;
+
+import static net.imglib2.roi.boundary.Boundary.StructuringElement.FOUR_CONNECTED;
```

Comment:
we use such a `StructuringElement` over and over again (e.g. `CCA` is algorithms). Should we put it into `imglib2` core?